src/main/java/gov/nasa/pds/tools/validate/BundleManager.java

```diff
@@ -452,7 +471,12 @@ public static Target getLatestBundle() {
             Crawler crawler = CrawlerFactory.newInstance(new File(dirName).toURI().toURL());
             IOFileFilter regexFileFilter = new RegexFileFilter(COLLECTION_LABEL_PATTERN);
             LOG.debug("findOtherBundleFiles:crawler {}",crawler);
-            allFiles = crawler.crawl(new File(dirName).toURI().toURL(),regexFileFilter);
+            //allFiles = crawler.crawl(new File(dirName).toURI().toURL(),regexFileFilter);
+            // Note: For some strange reason, the crawler goes into an infinite loop using the above call
+            //       so we will use an alternate call to get the list of files.
+            String[] extensions = new String[1];
```

Comment:
@qchaupds can we set this as a list constant and convert as needed to an array for this method?

same goes for the "collection" string below on line 479. i could imagine this being rough trying to track down in the future in the event they need to changed for whatever reason.

Reply:
@qchaupds also, in the future, can we just make sure we go through and "resolve" each of these comments as you complete them? i would think of the comments as a checklist. this way I know you saw them and made the fix. otherwise, I have to go back and look at my comments and see if it actually changes.